docs/ref/settings.txt

```diff
@@ -3001,9 +2995,7 @@ useful way to mitigate the risk of a client side script accessing the
 protected cookie data.
 
 Turning it on makes it less trivial for an attacker to escalate a cross-site
-scripting vulnerability into full hijacking of a user's session. There's not
```

Comment:
I don't see much advantage to removing the sentence here and in SESSION_COOKIE_SECURE. It eliminates the answer to the question, "is there a use case for disabling?" If the answer is "no", then perhaps there's an argument for removing the setting, but that's another matter.

Any objections? If not, I have a few other edits that I'll push along with this (mainly, some rewrapping closer to 79 characters for the modified lines).

Reply:
I think if we're going to recommend never turning this off we should be more direct than we were before. Without knowledge of any use-cases I'm in favour of deprecation and removal.